### PR TITLE
isnull filter needs boolean value for the Filter function

### PR DESCRIPTION
**Pull request by r0nni3**
On the generated API code

The issue:
/v1/objects/?query=col__isnull    ==> returns empty object
/v1/objects/?query=col__isnull:    ==> Error
/v1/objects/?query=col__isnull:true    ==> Error
/v1/objects/?query=col__isnull:false    ==> Error
/v1/objects/?query=col__isnull:0    ==> Error
/v1/objects/?query=col__isnull:1    ==> Error

With the proposed solution:
/v1/objects/?query=col__isnull    ==> returns empty object
/v1/objects/?query=col__isnull:    ==> rows for Filter("__isnull", false)
/v1/objects/?query=col__isnull:true    ==> rows for Filter("__isnull", true)
/v1/objects/?query=col__isnull:false    ==> rows for Filter("__isnull", false)
/v1/objects/?query=col__isnull:0    ==> rows for Filter("__isnull", false)
/v1/objects/?query=col__isnull:1    ==> rows for Filter("__isnull", true)

### CODE REVIEW

File: g_appcode.go [hunk 1]
```diff
@@ -1062,7 +1062,15 @@ func GetAll{{modelName}}(query map[string]string, fields []string, sortby []stri
 	for k, v := range query {
 		// rewrite dot-notation to Object__Attribute
 		k = strings.Replace(k, ".", "__", -1)
-		qs = qs.Filter(k, v)
+		if strings.Contains(k, "isnull") {
+			var v2 bool
+			if v == "true" || v == "1" {
+				v2 = true
+			}
+			qs = qs.Filter(k, v2)
```
Comment (amrfaissal) [inline g_appcode.go]: @r0nni3 The inner if statement can be replaced with: `qs = qs.Filter(k, (v == "true" || v == "1"))`

**Comment by amrfaissal** [comment]
@r0nni3 And It would be great if you create a branch (with your changes) based on the `develop` branch not `master`.

File: g_appcode.go (hunk 1 above)
Comment (r0nni3) [inline g_appcode.go]: @amrfaissal  Ok, im on it

**Comment by r0nni3** [comment]
@amrfaissal I will close this PR then and make a new PR

**Pull request closed without merging**